src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -510,4 +510,11 @@ public net.minecraft.world.level.levelgen.feature.trunkplacers.TrunkPlacerType <
 protected net.minecraft.world.level.portal.PortalForcer level # level
 public net.minecraft.world.level.storage.LevelResource <init>(Ljava/lang/String;)V # constructor
 private-f net.minecraft.world.level.storage.loot.LootPool rolls # rolls
-private-f net.minecraft.world.level.storage.loot.LootPool bonusRolls # bonusRolls
\ No newline at end of file
+private-f net.minecraft.world.level.storage.loot.LootPool bonusRolls # bonusRolls
+public net.minecraft.client.renderer.texture.atlas.sources.PalettedPermutations <init>(Ljava/util/List;Lnet/minecraft/resources/ResourceLocation;Ljava/util/Map;)V # constructor
```

Comment:
The AT entries should be in alphabetical order with respect to the rest of the entries and not at the end of the file